README.md

```diff
@@ -74,10 +74,18 @@ service.
 This repo uses submodules, so use `git clone --recursive`. You will also need to
 install the following:
```

Comment:
Maybe something like "this was successfully tested on..." instead of a hard you must have these versions. Ansible must be >2 fasho.... Also Python 2.7 must be your default system version or you must use a virtualenv.